tests/unit/test_workflow_files.py

```diff
@@ -1540,10 +1541,10 @@ def test_get_source_workflow_name(
         'cylc.flow.workflow_files.glbl_cfg',
         '''
         [install]
-            source dirs = /isla/nublar, /isla/sorna
+            source dirs = ~/isla/nublar, ~/isla/sorna
```

Comment:
Should test the env var case too
```suggestion
            source dirs = ~/isla/nublar, ${HOME}/isla/sorna
```